frontend/public/components/app.jsx

```diff
@@ -120,20 +120,28 @@ class App extends React.PureComponent {
             <Route path="/overview" exact component={NamespaceRedirect} />
 
-            <LazyRoute path="/catalog/all-namespaces" exact loader={() => import('./catalog/catalog-page' /* webpackChunkName: "catalog" */).then(m => m.CatalogPage)} />
-            <LazyRoute path="/catalog/ns/:ns" exact loader={() => import('./catalog/catalog-page' /* webpackChunkName: "catalog" */).then(m => m.CatalogPage)} />
-            <Route path="/catalog" exact component={NamespaceRedirect} />
-
             <LazyRoute path="/status/all-namespaces" exact loader={() => import('./cluster-overview' /* webpackChunkName: "cluster-overview" */).then(m => m.ClusterOverviewPage)} />
             <LazyRoute path="/status/ns/:ns" exact loader={() => import('./cluster-overview' /* webpackChunkName: "cluster-overview" */).then(m => m.ClusterOverviewPage)} />
             <Route path="/status" exact component={NamespaceRedirect} />
 
             <LazyRoute path="/cluster-health" exact loader={() => import('./cluster-health' /* webpackChunkName: "cluster-health" */).then(m => m.ClusterHealth)} />
             <LazyRoute path="/start-guide" exact loader={() => import('./start-guide' /* webpackChunkName: "start-guide" */).then(m => m.StartGuidePage)} />
 
-            {
-              // <LazyRoute path="/marketplace" exact loader={() => import('./marketplace/kubernetes-marketplace' /* webpackChunkName: "marketplace" */).then(m => m.MarketplacePage)} />
-            }
+            <LazyRoute path="/catalog/all-namespaces" exact loader={() => import('./catalog/catalog-page' /* webpackChunkName: "catalog" */).then(m => m.CatalogPage)} />
+            <LazyRoute path="/catalog/ns/:ns" exact loader={() => import('./catalog/catalog-page' /* webpackChunkName: "catalog" */).then(m => m.CatalogPage)} />
+            <Route path="/catalog" exact component={NamespaceRedirect} />
+
+            <LazyRoute path="/provisionedservices/all-namespaces" loader={() => import('./provisioned-services' /* webpackChunkName: "provisionedservices" */).then(m => m.ProvisionedServices)} />
+            <LazyRoute path="/provisionedservices/ns/:ns" loader={() => import('./provisioned-services' /* webpackChunkName: "provisionedservices" */).then(m => m.ProvisionedServicesPage)} />
+            <Route path="/provisionedservices" component={NamespaceRedirect} />
+
+            <LazyRoute path="/operators/all-namespaces" loader={() => import('./operators' /* webpackChunkName: "operators" */).then(m => m.OperatorsPage)} />
+            <LazyRoute path="/operators/ns/:ns" loader={() => import('./operators' /* webpackChunkName: "operators" */).then(m => m.OperatorsPage)} />
+            <Route path="/operators" component={NamespaceRedirect} />
+
+            <LazyRoute path="/marketplace" exact loader={() => import('./marketplace/kubernetes-marketplace' /* webpackChunkName: "marketplace" */).then(m => m.MarketplacePage)} />
```

Comment:
I think if the route is re-enabled you should re-enable the performance test [here](https://github.com/openshift/console/blob/12bf0f4ea98265c81a38589746bd426b7acecc35/frontend/integration-tests/tests/performance.scenario.ts#L35)